Automattic/simplenote-ios
# Textkit 2: Fixed gesture recognizer so tapping into text does not fallback to text kit 1

### Fix
In my previous PR with TextKit 2, #1690 , I added the basic setup for SPTextView to use Text Kit 2 if available, and I created a mechanism to fallback to text kit 1 without losing all formatting.  However in that PR if you did anything other than view a note you were immediately dropped back into Text Kit 1.  

In this PR I have updated gesture recognizer to not use `layoutManager`  so the view does not drop back to text kit 1.  This should allow you to tap into, interact with, edit, and save text while still in text kit 2.

Next up: Search

### Test
1. Select a note
2. Tap on any part of the text and start typing in it.  Do some basic editing of the note

- [x] confirm that you do not see `is switching to TextKit 1 compatibility mode because its layoutManager was accessed` in the console
- [x] Also smoke test iOS 16 or older to make sure the old processes still work

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> These changes do not require release notes.
